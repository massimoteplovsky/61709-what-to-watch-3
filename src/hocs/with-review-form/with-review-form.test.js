import React from "react";
import renderer from "react-test-renderer";
import withReviewForm from "./with-review-form.js";

const MockComponent = () => {
  return (
    <div></div>
  );
};

const MockComponentWrapped = withReviewForm(MockComponent);

it(`withReviewForm is rendered correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped/>
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
