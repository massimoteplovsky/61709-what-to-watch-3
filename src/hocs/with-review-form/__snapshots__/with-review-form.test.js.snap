// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withReviewForm is rendered correctly 1`] = `<div />`;
